configuration/monitor-acs.adoc

```diff
@@ -1,13 +1,47 @@
 [id="monitor-acs"]
-= Monitoring with Prometheus
+= Monitoring {product-title-short}
 include::modules/common-attributes.adoc[]
 :context: monitor-acs
 
 toc::[]
 
+== Monitoring with {osp}
+
+[role="_abstract"]
+link:https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html[{ocp}] includes a preconfigured,
+preinstalled, and self-updating monitoring stack that provides monitoring for core platform components.
+{product-title-short} exposes metrics to {osp} monitoring via an encrypted and authenticated endpoint.
+
+[IMPORTANT]
+====
+If you have previously configured monitoring with the Prometheus Operator,
+consider removing your custom ServiceMonitor resources. {product-title-short} ships with a pre-configured ServiceMonitor for {osp} monitoring.
+Multiple ServiceMonitors may result in duplicated scraping.
+====
+
+include::modules/enable-openshift-monitoring-central-operator.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/installing_ocp/install-central-config-options-ocp.adoc#install-central-config-options-ocp[Central configuration options using the Operator]
+
+include::modules/enable-openshift-monitoring-central-helm.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/installing_other/install-central-other.adoc#change-config-options-after-deployment-central-services_install-central-other[Changing configuration options after deploying the central-services Helm chart]
+
+== Monitoring with custom Prometheus
+
 [role="_abstract"]
 link:https://prometheus.io/[Prometheus] is an open-source monitoring and alerting platform.
-You can use it to monitor health and availability of Central and Sensor components of {rh-rhacs-first}. When you enable monitoring, {product-title-short} creates a new monitoring service on port number `9090` and a network policy allowing inbound connections to that port.
+You can use it to monitor health and availability of Central and Sensor components of {rh-rhacs-first}. When you enable monitoring, {product-title-short} creates a new monitoring service on port number 9090 and a network policy allowing inbound connections to that port.
+
+[NOTE]
+====
+This monitoring service exposes an endpoint that is not encrypted by TLS and has no authorization.
+Use this only when you do not want to use {osp} monitoring.
+====
 
 include::modules/enable-monitoring-central-operator.adoc[leveloffset=+1]
```

Comment:
This is actually still not resolved, sorry. This looks like an old version of the enable-openshift-monitoring-central-operator.adoc file, which is properly called on line 22. This file included here hasn't been updated and doesn't contain the latest changes. I think this include line should be deleted, and the file enable-monitoring-central-operator.adoc should be deleted.

Reply:
Ah, good catch. I added more context to it but I'm not sure if it's better now

![image](https://github.com/openshift/openshift-docs/assets/1616386/bb4f1ba8-9bc9-4d76-81f5-0ddc9d3cf6d0)

Reply:
maybe we should change the "old" names and add "custom" there ?

Reply:
I feel like we now have way too many sections. Seems very complicated for a feature that is essentially working out of the box without any interaction needed. Maybe make the `... by using` sub-sections?